### PR TITLE
Set workflow state to pending if no score in UI

### PR DESCRIPTION
**Pull request by westonkd**
_Closes PLAT-2303_
Test Plan:
- Create a report through the UI with no score.
- Verify the report's workflow state is set to 'pending' in Canvas.

### CODE REVIEW

**Comment by westonkd** [comment]
We need to first support this in Canvas.

File: app/models/originality_report.rb [hunk 1]
```diff
@@ -1,5 +1,4 @@
 class OriginalityReport < ApplicationRecord
-  validates :originality_score, :file_id, :tc_id, :submission, presence: true
-  validates :originality_score, inclusion: { in: 0..100, message: 'score must be between 0 and 100' }
```
Comment (abutterf) [inline app/models/originality_report.rb]: Are we banking on the Canvas api to enforce this for us?

File: db/migrate/20170421162639_change_originality_report_score_null.rb [hunk 2]
```diff
@@ -0,0 +1,5 @@
+class ChangeOriginalityReportScoreNull < ActiveRecord::Migration[5.0]
+  def change
+    change_column_null :originality_reports, :originality_score, true
```
Comment (abutterf) [inline db/migrate/20170421162639_change_originality_report_score_null.rb]: Should their be a corresponding file change to schema.rb for this migration?

File: app/models/originality_report.rb (hunk 1 above)
Comment (westonkd) [inline app/models/originality_report.rb]: Yes, we don't create the TP in our DB unless Canvas returns a successful response and Canvas does not return a successful response unless the score is nil or between 0..100

File: db/migrate/20170421162639_change_originality_report_score_null.rb (hunk 2 above)
Comment (westonkd) [inline db/migrate/20170421162639_change_originality_report_score_null.rb]: It looks like schema.rb already lacks a not null constraint for originality_score

**Pull request merged**